testsuite/src/it/java/io/quarkus/ts/startstop/SpecialCharsTest.java

```diff
@@ -176,6 +177,7 @@ public void spacesNative(TestInfo testInfo) throws IOException, InterruptedExcep
     }
 
     @Test
+    @Disabled("TODO: https://issues.redhat.com/browse/QUARKUS-4695")
```

Comment:
https://github.com/quarkusio/quarkus/issues/42248 should be referenced instead of product tracker